Yoast/wordpress-seo
# Remove sass

## Context


* This PR removes all .scss files and replaces them for .css files. The sass build step is also removed and replaced for some new steps.

## Summary


This PR can be summarized in the following changelog entry:

* Replaces sass with css.

## Relevant technical choices:

* A grunt copy task is added to copy the .css files from `src` to `dist` and to add the product version slug. 
* A Postcss task is added for adding the css files from node_modules.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and run `yarn`
* Run `grunt build`
* Make sure the correct files and filenames are added to the `dist` folder
* Make sure there are no visual changes in the plugin

**Criteria for merging to `trunk` as per Herre's Slack message:**
* "running both dev and production builds work as expected in regards to all styles, including creating a zip archive."

## UI changes

## Documentation

## Quality assurance


Fixes #
